spec/features/todo_items/todo_items_spec.rb

```diff
@@ -15,6 +15,7 @@
     } }
 
   before(:each) do
+    user.update_attribute('activation_token', nil)
```

Comment:
Trochę żmudne takie ustawianie `activation_token=nil` wszędzie prawda? Będzie znacznie gorzej w miarę rośnięcia funkcjonalności (pomijając już kwestie wydajnościowe). Poszukaj proszę metody jak to ładnie rozwiązać na poziomie fabryki - tak żeby od razu było to ustawiane na `nil`. Zazwyczaj się tego nie robi, ale znacznie nam to ułatwi pracę :)